enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -372,15 +390,15 @@ impl<'a> Handler<'a> {
         // array, measured in bytes.
         let mut offset = 0;
         for (t, mut u) in trusted.iter_mut().zip(untrusted.iter_mut()) {
-            u.base = ubuffer[offset..].as_mut_ptr();
-            u.size = t.size;
-            offset += t.size;
+            u.iov_base = ubuffer[offset..].as_mut_ptr() as *mut core::ffi::c_void;
```

Comment:
```suggestion
            u.iov_base = ubuffer[offset..].as_mut_ptr() as *mut _;
```

I think we can do this for all of the instances of `c_void` in this patch.